docs/index.rst

```diff
@@ -96,57 +96,523 @@ The final structural and functional annotation files are in GFF format and the s
    * - Directory/File Name
      - Description
    * - prefix_cath_funfam.gff
-     - gff functional annotation generated from Cath-FunFam (Functional Families) database
+     - GFF / tab-delimited functional annotation generated from Cath-FunFam (Functional Families) database
```

Comment:
This entire table some rows are italicized and some are not. I recommend not using italics for the entire table.